features/steps/fhir.py

```diff
@@ -116,6 +116,9 @@ def step_impl(context, field_name):
 
 @then('all references will resolve')
 def step_impl(context):
+    if context.vendor_skip:
+        return
+
```

Comment:
These two lines will need to be in each step that might be skippable. Not a fan of putting something in each step but couldn't find a handy way of skipping the step from within the "before_step" method. @erikwiffin - Any ideas?

Reply:
From what I can see, behave doesn't provide a way to skip an individual step like that. What about doing something with tags? Add a `healthcorp-skip` tag to anything that HealthCorp wants to skip, and then when the HealthCorp provider is chosen, dynamically add `--tags=-healthcorp-skip` at the start of the run.

Reply:
That's a good idea but I think building that eventual boolean statement of tags that Behave has to evaluate to determine which steps to run was getting out of hand with 2 decision drivers (User input, Use case). That's why I ended up manually implementing the check for tags for our different "use cases".